sdk/communication/Azure.Communication.PhoneNumbers/src/PhoneNumbersClient.cs

```diff
@@ -78,8 +80,28 @@ public PhoneNumbersClient(Uri endpoint, TokenCredential tokenCredential, PhoneNu
 
         #endregion
 
+        #region internal constructors
+        /// <summary> Initializes a new instance of PhoneNumbersClient. </summary>
+        /// <param name="clientDiagnostics"> The handler for diagnostic messaging in the client. </param>
+        /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
+        /// <param name="endpoint"> The communication resource, for example https://resourcename.communication.azure.com. </param>
+        /// <param name="apiVersion"> Api Version. </param>
+        internal PhoneNumbersClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint, string apiVersion = "2021-03-07")
```

Comment:
why do we need this constructor?

Reply:
It is in use by other constructor.

Reply:
It is used only by last private ctor, let's just move it there and delete this extra one.

Reply:
Moved